test/steps/configuration_steps.go

```diff
@@ -35,7 +35,7 @@ func ConfigurationSteps(suite *godog.Suite, fs *FeatureState) {
 		return nil
 	})
 
-	suite.Step(`^the global new-branch-push-flag configuration is set to false$`, func() error {
+	suite.Step(`^the global new-branch-push-flag configuration is false$`, func() error {
 		_ = fs.activeScenarioState.gitEnvironment.DeveloperRepo.Configuration(false).SetNewBranchPush(false, true)
```

Comment:
Don't know how old this is but the `SetNewBranchPush` function feels confusing since it has two boolean arguments. Would be nice to make it clearer from a call what the second one means. Going to think on that

Reply:
I agree, some level of stringly-typed code is occurring in a few places. I am keeping an eye on that as well.